### ISSUE TITLE
DESeq2: enable custom defined size factor

### ISSUE THREAD

**Issue by gallardoalba** [galaxyproject/tools-iuc#4425]
A user in Ghelp asked about the possibility of enabling custom-defined size factors: [Ghelp post](https://help.galaxyproject.org/t/how-to-set-up-user-defined-size-factor-in-deseq2/7593).

**Comment by Orange220** [comment]
Yes, enabling custom-defined size factor is useful in spike-in normalization experiments. It seems to haven't been added in the latest version (Galaxy Version 2.11.40.7+galaxy2). Looking forward to having it.

**Comment by Orange220** [comment]
To clarify, what we need to custom-defined, is not the"Method for estimateSizeFactors", but the value of the SizeFactor.
